recipe/meta.yaml

```diff
@@ -154,26 +97,19 @@ requirements:
     - mysql-devel                        # [not win]
     - postgresql                         # [not win]
     - zlib
+    - zstd
+    - zstd-static                        # [not win]
     - libxcb                             # [linux]
     - openssl
-    # For QDoc
-    # http://doc.qt.io/Qt-5/qdoc-guide-clang.html
-    # Only libclang is needed, but needs the full package for detection.
-    - llvmdev
-    - clangdev
-    - libclang
-    - libcups                            # [linux]
-    - zstd
-    - alsa-lib                           # [linux]
-    - krb5
+    - xorg-libxfixes                     # [linux]
+    - double-conversion
+    - pcre2
+    - dbus                               # [linux]
+    - glib
   run:
-    - {{ pin_compatible("nss") }}        # [unix]
-    - {{ pin_compatible("nspr") }}       # [unix]
+    - openssl                            # [not osx]
```

Comment:
I don't think this line is necessary. it will get picked up by `run_export`. same with libxcb below